spec/requestReply_at_operation_with-filter-per-operation.yaml

```diff
@@ -0,0 +1,66 @@
+channels:
+  root:
+    address: '/'
+    messages:
+      pingStatic:
+        $ref: '#/components/messages/pingStatic'
+      pingDynamic:
+        $ref: '#/components/messages/pingDynamic'
+      pong:
+        $ref: '#/components/messages/pong' 
+      dontWork:
+        $ref: '#/components/messages/genericError' 
+  pongDynamic:
+    address: null # will be used from ping.replyAddress
+    messages:
+      pong:
+        $ref: '#/components/messages/pong' 
+      dontWork:
+        $ref: '#/components/messages/genericError'
+
+
+operations:
+  requestStatic:
+    channel:
+      $ref: '#/channels/root'
+    messages:
+      - $ref: '#/channels/root/messages/pingStatic'
+    reply:
+      channel:
+        $ref: '#/channels/root'
+      messages:
+        - $ref: '#/channels/root/messages/pong'
+        - $ref: '#/channels/root/messages/genericError'
+  requestDynamic:
+    channel:
+      $ref: '#/channels/root'
+    messages:
+      - $ref: '#/channels/root/messages/pingDynamic'
+    reply:
+      address: # Attention: that will be applied to the messages of operations[x].channel
+        location: '$message.header#/replyTo'
```

Comment:
hehe, I wrote my other comment where I mention`addressLocation` before I saw this last file 😄 

so how about `addressLocation` instead of `address.location`? 
The advantage is that  it will not (collide with / confuse people) that see `address` on a channel level

Reply:
Or just straight up `location` instead of `addressLocation` 👍

Reply:
Just `location` might be insufficient to understand that we mean the channel address location. I bet for being more explicit when possible. `addressLocation` seems to be the optimum choice for me.

Reply:
So your want to habe correlationId having a description but the reply Adresse not? Or Remove the description form correlation id to make objects equal again?

Reply:
I think it makes sense to have a `description` field in `correlationId` because it gives you the opportunity to explain what kind of correlation information you're using, what kind of identifier, etc. However, I don't see any benefits of having a `description` field in this case. Do you?

Reply:
interesting, `description`, I did not take it into account that we can reuse the same object we have for `correlationId`. Actually it makes sense now to have `address.location`, I bet people would like to provide some info to `address.description` as they do with `correlationId`.

looking at Adeo and 
```
    costingCorrelationId:
      description: >
        This correlation ID is used for message tracing and messages
        correlation.
        This correlation ID is generated at runtime based on the `REQUEST_ID`
        and sent to the RESPONSE message.
      location: $message.header#/REQUEST_ID
```

for reply channel address people would probably also like to specify more details on how this header is generated and what is based on

Reply:
@GreenRover @jonaslagoni @fmvilas thoughts?

Reply:
I can't think of any use case. What do you have in mind, @derberg?

Reply:
@fmvilas sorry, I mean that basically `address.location` makes more sense than what I initially suggested - `addressLocation`

because of people taking advantage of `description` as it is in case of `correlationId`. And in the end `address.location` is also better because of the rule we have:

```
It's hard to know what the future brings; whenever possible, decisions should be made that allow for more options in the future. Sometimes this is unintuitive: spec rules often begin more strict than necessary with a future option to loosen when motivated by a real use case.
```